.github/workflows/benchmarks.yml

```diff
@@ -127,16 +145,14 @@ jobs:
         uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
         with:
           message: |
-            Nightly benchmarks failed!
+            ${{ github.event_name == 'schedule' && 'Nightly benchmarks failed!' || 'Benchmarks failed!' }}
             SDH Duty assignee, please have a look and follow this <https://github.com/elastic/observability-dev/blob/main/docs/apm/apm-server/runbooks/benchmarks.md|Runbook>!
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
-          slackChannel: "#apm-sever"
+          slackChannel: "#apm-server"
 
-      - name: Post to a Slack channel
-        id: slack
-        uses: slackapi/slack-github-action@v1.24.0
+      - uses: slackapi/slack-github-action@v1.24.0
```

Comment:
Does it makes sense to replace this GitHub Action by our [`slack-message`](https://github.com/elastic/apm-pipeline-library/blob/main/.github/actions/slack-message/action.yml) GitHub Action?

Reply:
The `slack-message` GitHub Action support the payload input if I remember correctly.

Reply:
Aaah, thanks. I kinda missed that.

https://github.com/elastic/apm-server/pull/11119/commits/6ea39be083318d38d4513a381201aa060ddf4960

Reply:
https://github.com/elastic/apm-server/pull/11119/commits/f5a7291b8760b92abc6b421c60ea0339a4aae73c

Reply:
https://github.com/elastic/apm-server/pull/11119/commits/55494d2b8190aee6f21598fd905d23abf6c633ba

Reply:
slack-message support custom payload, any that you can build with the [Block kit](https://api.slack.com/block-kit)